core/kazoo_bindings/src/kazoo_bindings.erl

```diff
@@ -647,27 +645,28 @@ apply_responder(#kz_responder{module=M
 
 -spec log_undefined(atom(), atom(), non_neg_integer(), list()) -> 'ok'.
 log_undefined(M, F, Length, [{M, F, _Args,_}|_]) ->
-    lager:debug("undefined function ~s:~s/~b", [M, F, Length]);
+    lager:error("undefined function ~s:~s/~b", [M, F, Length]);
```

Comment:
I agree for `log_undefined`, however `log_function_clause` only has true positives right?